tutorial/tutorial.md

```diff
@@ -0,0 +1,227 @@
+# Tutorial
+## Introduction
+FalCAuN is a toolkit for testing black-box systems like CPSs (cyber-physical systems) based on automata learning and model checking.
+Given specifications as STL(Signal Temporal Logic) formulas,
+FalCAuN checks the given system satisfies the specifications by finding an input signal that its corresponding output signal violates the specifications.
+It is called falsification.
+
+The main feature of FalCAuN is the combination of BBC and optimization-based falsification.
+
+BBC (Black-box checking) [Meinke, MLDSA'18], [Peled et al., JALC'02] is a way of testing method that constructs a mealy machine by automata-learning from a black-box system and
```

Comment:
"mealy" must be "Mealy" since it is named after a name of a person (https://en.wikipedia.org/wiki/Mealy_machine)

Reply:
That's true. Reflected it.